ts/testUtils.ts

```diff
@@ -56,14 +63,37 @@ export const assertStderr = (output: Output, expected: string) => {
   }
 };
 
+export const getStorePath = (flakeDir: string, attribute: string): string => {
```

Comment:
This is a bit different than what we've been doing so far, which is that building always happens at the Haskell layer, and here we just generate statically a flake.nix file. 

If we were to continue in that direction, you can still refer to the output path, but not e.g. compare it for equality in Typescript. In particular, you could have `getStorePath : Package -> NixExpression`, for example (and similarly for `Environment` and `Check`.

Anyhow, I think this is *philosophically* a big change, so maybe it'd be helpful to know what the intended use-case is.

@soenkehahn @alexdavid are maybe more qualified to speak to all this, though.

Reply:
Ah, if this is for testing, never mind! Sorry for the noise!